tests/scaletest/scale_test.go

```diff
@@ -262,6 +277,13 @@ func CheckReboot(t *testing.T, wg *sync.WaitGroup) {
 		}
 		json.Unmarshal(byteValue, &testbedParams)
 		go Reboot(t, wg, KUBENODE, testbedParams.AkoParam.Clusters[0].KubeNodes[0].IP, testbedParams.AkoParam.Clusters[0].KubeNodes[0].UserName, testbedParams.AkoParam.Clusters[0].KubeNodes[0].Password, 0)
+		// Disable swap on the rebooted node
+		g := gomega.NewGomegaWithT(t)
+		time.Sleep(20 * time.Second)
```

Comment:
Any reason we have a 20s sleep (before doing remote execute) here when we already have the 20s interval for `Eventually` too?